services/iam/app/models/root.rb

```diff
@@ -6,7 +6,7 @@ class Root < Iam::ApplicationRecord
   # has_many :ssh_keys
 
   def to_urn
-    "#{self.class.urn_base}:#{tenant.account_id}:root/#{id}"
```

Comment:
In which case is there a root without a tenant?

Reply:
Yes, when creating a new root, there is no tenant associated yet.